js/models/conversations.js

```diff
@@ -2064,18 +2065,9 @@
       return {
         server: this.get('server'),
         channelId: this.get('channelId'),
+        conversationId: this.get('id'),
```

Comment:
Na they're different. `conversationId` is a local id assigned to the conversation, where as `server` and `channelId` are ids that relate a conversation to a specific channel on a server

Reply:
While the serverURL + channelId will be unique to a conversation, the issue is that we can't retrieve the conversation with that data; only by the conversationId

Reply:
Oh right
My comment was actually wrong as well
After looking at the code again the reason we are passing these 3 is because we are using these functions
```
      const { server, channelId, conversationId } = channelSettings;
      const serverAPI = lokiPublicChatAPI.findOrCreateServer(server);
...
      const channelAPI = serverAPI.findOrCreateChannel(
        channelId,
        conversationId
      );
```
I think I can clean this up in the same commit addressing the comments below